app/service/rest_svc.py

```diff
@@ -3,12 +3,15 @@
 import glob
 import os
 import pathlib
+import pptx
 import uuid
 import yaml
 
 from collections import defaultdict
 from datetime import time
 
```

Comment:
https://www.python.org/dev/peps/pep-0008/#imports can you re arrange the imports?

Reply:
This branch is way out of date & I haven't touched it in a while... but yes